packages/@react-stately/layout/src/TableLayout.ts

```diff
@@ -53,6 +53,9 @@ export class TableLayout<T> extends ListLayout<T> {
     for (let column of this.collection.columns) {
       let props = column.props as ColumnProps<T>;
       let width = props.width ?? props.defaultWidth;
```

Comment:
Idea would be that this would turn into `let width = props.width ?? props.defaultWidth ?? options.getDefaultWidth(column)`